src/pages/AdminPage2.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import { Container, Heading, Flex, Text } from '@chakra-ui/react';
+import { ChevronDownIcon, ChevronUpIcon } from '@chakra-ui/icons';
+import GenerateReportModal from '../components/AdminPageTable/GenerateReportModal';
+import ExportLogsModal from '../components/AdminPageTable/ExportLogsModal';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ExportLogsModal' is defined but never used.